### DISCUSSION TITLE
interacting with the db in nodejs

### DISCUSSION THREAD

**Discussion by bkyerv** [planetscale/discussion#313] [General]
When looking for ways to interact with the db all I was able to find is using either mysql cli or prisma. Is there any other way to interact with the db in a simple expressjs app but not using prisma? maybe something like node-pg?

**Comment by fooforge** [comment]
Hey @bkyerv, you can simply use the [node-mysql2](https://github.com/sidorares/node-mysql2) module. Check out [our documentation](https://planetscale.com/docs/tutorials/connect-nodejs-app) where we walk you through connecting a Node.js app with PlanetScale and [this blog post](https://planetscale.com/blog/create-a-harry-potter-api-with-node-js-express-mysql-and-planetscale) for another example.
